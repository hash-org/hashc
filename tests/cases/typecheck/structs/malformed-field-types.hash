// run=fail


Foo := struct(bar: Type)
