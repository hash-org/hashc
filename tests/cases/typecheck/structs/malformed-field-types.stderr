error[0020]: types mismatch, wanted `Type`, but got `Trait`
 --> $DIR/malformed-field-types.hash:4:20
3 |   
4 |   Foo := struct(bar: Type)
  |                      ^^^^ ...but this is of type `Trait`
